backend/scripts/verify-ci.sh

```
@@ -0,0 +1,219 @@
+check_module "archiving"
+check_module "tournament"
+check_module "api"
+
+echo ""
+echo "🧪 Running unit tests..."
+
+# Run tests for each module
+test_module() {
+    local module=$1
+    echo "Testing $module..."
+    if cargo test -p $module; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)